src/framework/core/js/Fluid.js

```diff
@@ -1851,6 +1851,9 @@ var fluid = fluid || fluid_3_0_0; // eslint-disable-line no-redeclare
 
     // A function to tag the types of all Fluid components
     fluid.componentConstructor = function () {};
+    Object.defineProperty(fluid.componentConstructor, "name", {
```

Comment:
Explain why we did this because I already barely remember!

Reply:
Looks like the reason was so that we can detect an Infusion component in src/framework/core/js/jquery.keyboard-a11y.js without having to depend on Fluid.js